Makefile.js

```diff
@@ -835,7 +835,7 @@ target.checkRuleFiles = function() {
             }
 
             // check eslint:recommended
-            const recommended = require("./conf/eslint-recommended");
+            const recommended = require("./packages/js").configs.recommended;
```

Comment:
We should also update the paths in the two error messages below:

https://github.com/eslint/eslint/blob/e05de4a0943e9f8a2a85eabc6f7bd5fe3012f718/Makefile.js#L842

https://github.com/eslint/eslint/blob/e05de4a0943e9f8a2a85eabc6f7bd5fe3012f718/Makefile.js#L847

`./conf/eslint-recommended.js` -> `./packages/js/src/configs/eslint-recommended.js`